### PR TITLE
Add Dogechain network

### PR DESCRIPTION
**Pull request by yodedex**
Fixes # .

Changes proposed in this pull request:
-  Add Dogechain
- 
-

### CODE REVIEW

File: src/networks.json [hunk 1]
```diff
@@ -980,6 +980,21 @@
     "start": 0,
     "logo": "ipfs://QmbENgHTymTUUArX5MZ2XXH69WGenirU3oamkRD448hYdz"
   },
+  "2000": {
+    "key": "2000",
+    "name": "Doge Chain",
+    "shortName": "dogechain",
+    "chainId": 2000,
+    "network": "mainnet",
+    "testnet": false,
```
Comment (ChaituVR) [inline src/networks.json:989]:
```suggestion
```

File: src/networks.json [hunk 2]
```diff
@@ -980,6 +980,21 @@
...
+    "multicall": "0x6f9D3f3932B417bd4957585D236Cbc32b32C0BDc",
+    "rpc": [
+      "https://rpc.dogechain.dog"
+    ],
+    "explorer": "https://explorer.dogechain.dog",
+    "start": 0,
```
Comment (ChaituVR) [inline src/networks.json:995]:
```suggestion
    "start": 877115,
```

File: src/networks.json [hunk 3]
```diff
@@ -980,6 +980,21 @@
...
+    "logo": "https://raw.githubusercontent.com/dogechain-lab/docs/main/symbol_var_6.png"
```
Comment (ChaituVR) [inline src/networks.json:996]: Can upload and pin it to IPFS plz :) To maintain consistency with other networks

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/networks.json (hunk 3 above)
Comment (MrCuddleMuffin) [reply src/networks.json:995]: 995 "logo": "ipfs://bafkreigovfh3pinsdih777issfgaflwu2yjzroljs2642gbvwikcd3nm4i
"

File: src/networks.json [hunk 4]
```diff
@@ -980,6 +980,20 @@
...
+    "shortName": "dogechain",
+    "chainId": 2000,
+    "network": "mainnet",
+    "multicall": "0x6f9D3f3932B417bd4957585D236Cbc32b32C0BDc",
+    "rpc": [
+      "https://rpc.dogechain.dog"
+    ],
+    "explorer": "https://explorer.dogechain.dog",
+    "start": 877115,
+    "logo": "https://raw.githubusercontent.com/dogechain-lab/docs/main/symbol_var_6.png"
```
Comment (ChaituVR) [inline src/networks.json:995]:
```suggestion
    "logo": "ipfs://bafkreigovfh3pinsdih777issfgaflwu2yjzroljs2642gbvwikcd3nm4i"
```

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**